### PR TITLE
Proposed changes to first draft

### PR DESCRIPTION
**Pull request by r12a**

### CODE REVIEW

**Review by aphillips — APPROVED** [review]

**Comment by aphillips** [comment]
Hi Richard, you didn't give me write access to the new repo, so I can't merge the PR. Please to fix?

**Comment by r12a** [comment]
Hm, had a bit of trouble getting it to stick, but hopefully it worked this time.  Could you try again and let me know if you had success?

**Comment by aphillips** [comment]
Woot! It works.

**Pull request merged**
